query/queries_and_views.ipynb

```diff
@@ -854,6 +859,2063 @@
+       "      <th>active</th>\n",
+       "      <th>organization_id</th>\n",
+       "      <th>system</th>\n",
+       "      <th>code</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>2588</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>http://nucc.org/provider-taxonomy</td>\n",
+       "      <td>208D00000X</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>2624</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>http://nucc.org/provider-taxonomy</td>\n",
+       "      <td>208D00000X</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2682</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>http://nucc.org/provider-taxonomy</td>\n",
+       "      <td>208D00000X</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>2688</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>http://nucc.org/provider-taxonomy</td>\n",
+       "      <td>208D00000X</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>2702</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "      <td>http://nucc.org/provider-taxonomy</td>\n",
+       "      <td>208D00000X</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  pr_id practitioner_id active organization_id  \\\n",
+       "0  2588            None   None            None   \n",
+       "1  2624            None   None            None   \n",
+       "2  2682            None   None            None   \n",
+       "3  2688            None   None            None   \n",
+       "4  2702            None   None            None   \n",
+       "\n",
+       "                              system        code  \n",
+       "0  http://nucc.org/provider-taxonomy  208D00000X  \n",
+       "1  http://nucc.org/provider-taxonomy  208D00000X  \n",
+       "2  http://nucc.org/provider-taxonomy  208D00000X  \n",
+       "3  http://nucc.org/provider-taxonomy  208D00000X  \n",
+       "4  http://nucc.org/provider-taxonomy  208D00000X  "
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "with query_engine.connect() as con:\n",
+    "    con.execute(\"\"\"\n",
+    "      CREATE OR REPLACE VIEW flat_practitioner_role AS\n",
+    "      SELECT P.id AS pr_id, P.practitioner.practitionerId as practitioner_id,\n",
```

Comment:
For PractitionerRole, can we add:

* speciality
* location
* healthcareService -> this would require a new view for this Resource type

Reply:
Done; not sure why you said a new view is required for `healthcareService`; please let me know if this is not what you wanted.